src/components/asset-list/RecyclerAssetList2/FastComponents/FastCurrencySelectionRow.tsx

```diff
@@ -0,0 +1,288 @@
+import React from 'react';
+import { Text as RNText, StyleSheet, View } from 'react-native';
+import {
+  // @ts-ignore
+  IS_TESTING,
+} from 'react-native-dotenv';
+// @ts-ignore
+import { ContextMenuButton } from 'react-native-ios-context-menu';
+import RadialGradient from 'react-native-radial-gradient';
+import fonts from '../../../../styles/fonts';
+import { ButtonPressAnimation } from '../../../animations';
+import { CoinRowHeight } from '../../../coin-row';
+import FastCoinIcon from './FastCoinIcon';
+import { Text } from '@rainbow-me/design-system';
+import { useAccountAsset } from '@rainbow-me/hooks';
+import { borders, colors, fontWithWidth, padding } from '@rainbow-me/styles';
+import { isETH } from '@rainbow-me/utils';
+
+const SafeRadialGradient = (IS_TESTING === 'true'
+  ? View
+  : RadialGradient) as typeof RadialGradient;
+
+export default React.memo(function FastCurrencySelectionRow({
+  item: {
+    uniqueId,
+    showBalance,
+    showFavoriteButton,
+    showAddButton,
+    onPress,
+    theme,
+    nativeCurrency,
+    nativeCurrencySymbol,
+    favorite,
+    toggleFavorite,
+    contextMenuProps,
+    symbol,
+    address,
+    name,
+    testID,
+  },
+}: {
+  item: any;
+}) {
+  const { isDarkMode } = theme;
+
+  // TODO
```

Comment:
Would be cool to at least add what this TODO is about and if you feel like it, maybe even link a Linear issue with it